Player.java

```diff
@@ -0,0 +1,29 @@
+package TicTacToe;
+
+class Player {
```

Comment:
You forgot the "public" modifier - as I said in the class: for now, only public classes are permitted

Reply:
True, we haven't learned package-level entry yet.